lib/ReactGridLayout.jsx

```diff
@@ -201,13 +258,20 @@ var ReactGridLayout = React.createClass({
     var activeDrag = {
       w: w, h: h, x: l.x, y: l.y, placeholder: true, i: i
     };
+
+    activeDrag = this.props.onResize(layout, layout, l, activeDrag, {e, element, size}) || activeDrag;
```

Comment:
Well onResizeStart/onDragStart you ideally would be canceling with preventDefault so no return necessary.
Ideally if there were a way to set both full layout/active drag layout items that would be ideal.
I'm attempting to implement a lot of restrictions around dragging/resizing  so controlling the full layout/actively dragging/resizing item is what I need.

I realize I can modify the layout and activeDrag by just simple object reference but that's just bad. Besides calling a setState of ReactGridLayout and passing down new props layout during the drag (which would probably be very inefficient and cause some weird UI states temporarily) I'm not sure what else is the solution here.

Basically how would you allow the user to control the positioning of the entire layout/actively dragging item while it's happening?